tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -555,7 +556,9 @@ class DxcIncludeHandlerForInjectedSources : public IDxcIncludeHandler {
     _COM_Outptr_result_maybenull_ IDxcBlob **ppIncludeSource
   ) override {
     try {
-      *ppIncludeSource = includeFiles.at(std::wstring(pFilename));
+      std::wstring FilenameStr(pFilename);
+      std::replace(FilenameStr.begin(), FilenameStr.end(), '/', '\\');
```

Comment:
This looks like a hidden dependency between the code that saves the file by translating everything to back slashes. Can we make that dependency explicit? For example by adding a `NormalizePath()` function or something.